cmd/minikube/cmd/start.go

```diff
@@ -672,7 +672,8 @@ func configureRuntimes(runner cruntime.CommandRunner) cruntime.Manager {
 		exit.WithError(fmt.Sprintf("Failed runtime for %+v", config), err)
 	}
 
-	err = cr.Enable()
+	disableOtherContainerEngines := viper.GetString(vmDriver) != constants.DriverNone
```

Comment:
While this is clever, let's aim for easier readability:

```
disableOthers := true
if  viper.GetString(vmDriver) == constants.DriverNone {
  disableOthers = false
}
cr.Enable(disableOthers)
```

Also, prefer shorter variable names for variables which have a very short scope.